rust/ql/src/queries/telemetry/RustAnalyzerComparison.qll

```diff
@@ -9,12 +9,13 @@ import rust
 
 pragma[nomagic]
 private predicate resolvesAsItem(Resolvable r, Item i) {
-  r.getResolvedPath() = i.getExtendedCanonicalPath() and
-  (
-    r.getResolvedCrateOrigin() = i.getCrateOrigin()
-    or
-    not r.hasResolvedCrateOrigin() and not i.hasCrateOrigin()
-  )
+  none()
+  // r.getResolvedPath() = i.getExtendedCanonicalPath() and
+  // (
+  //   r.getResolvedCrateOrigin() = i.getCrateOrigin()
+  //   or
+  //   not r.hasResolvedCrateOrigin() and not i.hasCrateOrigin()
+  // )
```

Comment:
What is `RustAnalyzerComparison.qll`?  Is there a reason for leaving these predicates in (as `none()`) for the time being?

Reply:
It was used to compare data in DCA, but those reports have now been disabled.

Reply:
Great.